src/components/index/page-sections/StartupSection.jsx

```diff
@@ -0,0 +1,70 @@
+import { Link } from '@reach/router';
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import PlanetImage from '../../../images/planet.png';
+// import SectionHeading from '../section-heading/SectionHeading';
+
+const NewsSection = () => {
+  return (
+    // Section Content Container
+    <div className="container mx-auto mb-40">
+      {/* Section Header */}
+      <h1 className="text-5xl mb-12">
+        <FormattedMessage id="index.startups.title" />
+      </h1>
+      {/* 2 ROW GRID - FOR 2 NEWS ARTICLES  */}
+      <div className="grid grid-cols-2 grid-flow-row gap-10">
```

Comment:
Wenn möglich das grid etc. auch in eigene Komponente, das wiederholt sich ja schon mehrfach.